src/lib.rs

```diff
@@ -37,8 +42,9 @@ impl Bundle {
                 Ok(resource) => resource,
                 Err(error) => {
                     if strict {
-                        return Err(PyValueError::new_err(format!(
-                            "{error:?} - Fluent file contains errors"
```

Comment:
The debug output of the error class was _really_ long and difficult to read. Ideally we would have an `errors` attribute on the Python exception which contains each error as structured data.